lib/lte_link_control/lte_lc_helpers.c

```diff
@@ -815,6 +818,62 @@ int parse_cereg(const char *at_response,
 		}
 	}
 
+	/* Check PSM parameters only if we are connected */
+	if ((*reg_status != LTE_LC_NW_REG_REGISTERED_HOME) &&
+	    (*reg_status != LTE_LC_NW_REG_REGISTERED_ROAMING)) {
+		goto clean_exit;
+	}
+
+	if (psm_cfg != NULL) {
+		char active_time_str[9] = {0};
+		char tau_ext_str[9] = {0};
+		int str_len = 8;
+		int err_active_time;
+		int err_tau;
+
+		psm_cfg->active_time = -1;
+		psm_cfg->tau = -1;
+
+		/* Get active time */
+		err_active_time = at_params_string_get(
+				&resp_list,
+				is_notif ? AT_CEREG_ACTIVE_TIME_INDEX :
+					   AT_CEREG_READ_ACTIVE_TIME_INDEX,
+				active_time_str, &str_len);
+		if (err_active_time) {
+			LOG_DBG("Active time not found, error code: %d", err_active_time);
```

Comment:
To be consistent with other error traces.
```suggestion
			LOG_DBG("Active time not found, error: %d", err_active_time);
```